packages/gnome_session.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Gnome_session < Package
+class Gnome_session < Meson
   description 'The GNOME Session Handler'
   homepage 'https://gitlab.gnome.org/GNOME/gnome-session'
   version '43.0'
```

Comment:
Update to [45.0](https://gitlab.gnome.org/GNOME/gnome-session/-/tags/45.0)?

Reply:
Gnome_session now has a hard systems dep, so deprecating.